### ISSUE TITLE
jpan ja webpage inline-notes

### ISSUE THREAD

**Issue by xfq** [w3c/type-samples#85]
![71465768-6ea68180-27f8-11ea-9083-b6b8db9fd84c](https://user-images.githubusercontent.com/4839211/71974365-3d22b280-3209-11ea-8191-d9d61f07ee1b.png)

---
tag: inline-notes Ruby annotation using ideographic characters.
source: [国語の文法](https://www.kokugobunpou.com/文の組み立て/連文節/)
----

**Comment by xfq** [comment]
This seems more like interlinear annotations (rather than inline notes) to me, because it is not text inserted in the line itself (like warichu), but text attached to a certain word or phrase and aligned to the corresponding base text which it annotates.

**Comment by xfq** [comment]
Perhaps interlinear annotations can be also seen as a kind of inline notes?

**Comment by r12a** [comment]
Yes. Remember that the tags used here relate to the Language Enablement Index sections. If you look at https://w3c.github.io/typography/#inline_notes (which you should) you'll see that the section title is Inline notes & annotations. So that covers ruby and warichu (but doesn't cover sidenotes, footnotes, endnotes, etc - which are addressed further down).

**Comment by himorin** [comment]
Not on labels and categorize, but for use cases, recently there are many non-Ruby use cases using Ruby like sample above - to label some part of text using Ruby text as label. Also,

- translation of conversation, such that Japanese translation of non-Japanese conversation section in Japanese book
- another way of readings (or voice in the heart) for specific keywords - like https://toaru-project.com/index_3/core_sys/images/main/common/logo.png
